cli.js

```diff
@@ -35,10 +35,10 @@ function whatBump(commits) {
     if (commit.notes.length > 0) {
       shouldRelease = true;
       major++;
-    } else if (commit.type === "feat") {
+    } else if (commit.type === "feat" || "perf") {
       shouldRelease = true;
       minor++;
-    } else if (commit.type === "fix") {
+    } else if (commit.type === "build" ||  "chore" || "ci" || "docs" || "fix" || "style" || "refactor" || "test") {
```

Comment:
the idea in this minimal module is to prevent releases for build, chore, ci, docs, style, refactoring, and test changes. Meaning those changes do not require a new version to be published. I think it does not make sense to publish a package for `build`, `chore`, `ci`, `docs`, `style`, `test` or `refactor`, ... changes

Reply:
I've removed these additional type to be inline with the intended design.